metals/src/main/scala/scala/meta/internal/metals/codeactions/ImportMissingSymbol.scala

```diff
@@ -60,4 +89,6 @@ object ImportMissingSymbol {
   def title(name: String, packageName: String): String =
     s"Import '$name' from package '$packageName'"
 
+  def allSymbolsTitle: String =
+    s"Import all missing symbols that are unambiguous"
```

Comment:
I'm curious if showing the `unambiguous` is a bit verbose to the user.
```suggestion
    s"Import all possible missing symbols"
```
What do you think?